save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/basic/FiltersRow.kt

```diff
@@ -19,86 +21,144 @@ import react.dom.html.ReactHTML.option
 import react.dom.html.ReactHTML.select
 import react.useState
 
+val testExecutionFiltersRow = testExecutionFiltersRow()
```

Comment:
Maybe make `testExecutionFiltersRow()` private in it's file and declare this `val` there but not in `FiltersRow`

Reply:
oh, nevermind, my bad